rviz_rendering/include/rviz_rendering/point_cloud.hpp

```diff
@@ -58,31 +58,39 @@ class RenderSystem;
 class Matrix4;
 }
 
-namespace rviz
+namespace rviz_rendering
 {
 
 class PointCloud;
 class PointCloudRenderable : public Ogre::SimpleRenderable
 {
 public:
-  PointCloudRenderable(PointCloud* parent, int num_points, bool use_tex_coords);
+  PointCloudRenderable(PointCloud * parent, int num_points, bool use_tex_coords);
   ~PointCloudRenderable();
 
-  Ogre::RenderOperation* getRenderOperation() { return &mRenderOp; }
+#ifdef __clang__
+# pragma clang diagnostic push
+# pragma clang diagnostic ignored "-Woverloaded-virtual"
+#endif
+  Ogre::RenderOperation * getRenderOperation() {return &mRenderOp;}
+#ifdef __clang__
+# pragma clang diagnostic pop
+#endif
```

Comment:
This looks like a warning you should not suppress, it looks like the signature has changed. Might be something new with 1.10.

Reply:
This warning is about name-hiding. There is (and always has been) a method `void getRenderOperation(Ogre::RenderOperation &)` in `Ogre::SimpleRenderable`. Since the name of the method is the same, this overloading here hides the original function. It seems that this has always been the case, but neither gcc nor msvc complain about it.

Reply:
Ok, sounds like a false positive then 👍.

At best we could let the Ogre guys know that they should use a different name for virtual functions.